packages/storybook/stories/Setup.stories.mdx

```diff
@@ -115,12 +114,11 @@ npm install @azure/communication-calling@1.6.1-beta.1
 ### Testing your app
 
 You need to downgrade or consider removing the provided `App.test.tsx` file. This is because it is
-using the new testing library `@testing-library/react` which is not compatible with other versions of
-react. If this file is left alone the app will not build correctly in the next step.
+using the new testing library `@testing-library/react` which is not compatible with versions of react `< 18`. If this file is left alone the app will not build correctly in the next step.
```

Comment:
with react versions ` < 18`. If this file is not modified, the app will fail to build correctly in the next step.